jvm-packages/xgboost4j/src/main/java/ml/dmlc/xgboost4j/java/Booster.java

```diff
@@ -85,11 +85,26 @@ static Booster loadModel(InputStream in) throws XGBoostError, IOException {
       os.write(buf, 0, size);
     }
     in.close();
-    Booster ret = new Booster(new HashMap<String, Object>(), new DMatrix[0]);
```

Comment:
maybe we can delete 

``` java
static Booster loadModel(InputStream in) throws XGBoostError, IOException {
```
and move the logic (getting buffer from InputStream) into XGBoost.java

Reply:
done